src/DynamoCore/Engine/CodeCompletion/CodeCompletionServices.cs

```diff
@@ -69,7 +69,7 @@ internal IEnumerable<CompletionData> GetCompletionsOnType(string code, string st
                 }
             }
 
-            return members.Select(x => CompletionData.ConvertMirrorToCompletionData(x));
+            return members?.Select(x => CompletionData.ConvertMirrorToCompletionData(x));
```

Comment:
Adding null check.

Reply:
awesome, this will make debugging so much easier!